changelog/fragments/update-kb-from-36124ae2e027-to-9c02d557f01b.yaml

```
@@ -0,0 +1,19 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      For Golang based-operators, add the command `òperator-sdk edit` which allows users edit the project layout to support Multi-Group.
```

Going forward, wdyt about using plugin key to denote which plugin was modified, ex
```suggestion
      (go/v2) Added the command `òperator-sdk edit` which allows users edit the project layout to support Multi-Group.
```